json-path/src/main/java/com/jayway/jsonpath/Predicate.java

```diff
@@ -23,6 +24,8 @@
 
     boolean apply(PredicateContext ctx);
 
+    boolean check(TokenStack stack, int idx);
```

Comment:
I want Predicate to be a functional interface. This will not work well with java 8.

Reply:
So funny to read this after all those years, because his project 'vjdbc' has been forked here on GitHub... :D
https://github.com/daitangio/vjdbc